shoop/admin/templates/shoop/admin/products/_edit_base_form.jinja

```diff
@@ -15,7 +15,11 @@
     {{ bs3.field(product_form.barcode) }}
     {{ bs3.field(product_form.gtin) }}
     {{ bs3.field(product_form.category) }}
-    {{ bs3.field(product_form.stock_behavior) }}
```

Comment:
I think i'll go with the comment route because it's more clear to show just a text instead of select that is not actually selectable. It would annoy the hell out of me to always click the dropdown and try to find out why there are no other options available and will there be etc.